api-tester/src/main/java/com/revenuecat/apitester/kotlin/StoreProductAPI.kt

```diff
@@ -39,6 +39,7 @@ private class StoreProductAPI {
         check(googleStoreProduct)
         val productDetails: ProductDetails = googleStoreProduct.productDetails
         val subscriptionOptions: List<GoogleSubscriptionOption> = googleStoreProduct.subscriptionOptions
+        val defaultOption: GoogleSubscriptionOption? = googleStoreProduct.defaultOption
```

Comment:
I think this would be covered by the API tests of `StoreProduct` above? Since we call `check` here we are making sure it has the same interface. I think we should check GoogleStoreProduct-only fields here.

Reply:
here i was checking that it's the more specific `GoogleSubscriptionOption` type